server/applications.json

```diff
@@ -1166,6 +1166,109 @@
                 }
             ]
         },
+        "unity": {
+            "enabled": true,
+            "host_name": "unity",
+            "environment": "{}",
+            "variants": [
+                {
+                    "name": "6000.0.32f1",
+                    "label": "6.0",
+                    "executables": {
+                        "windows": [
+                            "C:\\Program Files\\Unity\\Hub\\Editor\\6000.0.32f1\\Editor\\Unity.exe"
+                        ],
+                        "darwin": [
+                            "/Applications/Unity/Hub/Editor/6000.0.32f1/Unity.app"
+                        ],
+                        "linux": []
+                    },
+                    "arguments": {
+                        "windows": [],
+                        "darwin": [],
+                        "linux": []
+                    },
+                    "environment": "{}"
+                },
+                {
+                    "name": "2021-3",
+                    "label": "2021.3.5f1",
+                    "executables": {
+                        "windows": [
+                            "C:\\Program Files\\Unity\\Hub\\Editor\\2021.3.5f1\\Editor\\Unity.exe"
+                        ],
+                        "darwin": [
+                            "/Applications/Unity/Hub/Editor/2021.3.5f1/Unity.app"
+                        ],
+                        "linux": []
+                    },
+                    "arguments": {
+                        "windows": [],
+                        "darwin": [],
+                        "linux": []
+                    },
+                    "environment": "{}"
+                }
+            ]
+        },
+        "unity": {
```

Comment:
Something is wrong with the structure, this key is defined twice here?

Reply:
It was a wrong folder anyway.